csuntechlab/iSTART
# STR-209 -- Wire up dashboard to proper participant user_group.

# Description
***Branches off Luis's STR-300-hack***.
Added functionality to redirect to proper dashboard user_group.
  
# Testing
1. Login to control group user.
- Ensure the control group dashboard has no modules + card message.
2. Login to comparison group user.
- Ensure comparison group dashboard has 1 module (*comparison module*)
3. Login to intervention group user.
- Ensure intervention group dashboard has all 5 modules.

**IMPORTANT**
Don't merge in yet until Luis says so.
